### ISSUE TITLE
GPU proxy

### ISSUE THREAD

**Issue by techtonik** [p3/regal#164]
Hi Nigel,

How do you think - is it possible to proxy GPU from remote server to be available as local graphic card? The problem is that GPU on my notebook is burned out and I still need to work with OpenGL applications here. But I also have pretty beefy desktop in the same network and I thought maybe it it possible to redirect some stuff for the processing there?

**Comment by nigels-com** [comment]
Yes, I think that's possible in principle.  But I see that as more of a R&D project than something production ready.  The ApiTrace functionality ought to be suitable for this, but might need to be synced up the current version of ApiTrace, and modifications for playback over a network socket.

**Comment by techtonik** [comment]
Thanks. It is unlikely that I will be able to implement this myself, so I've created the issue at https://github.com/apitrace/apitrace/issues/403
